BibBuddy/app/src/main/java/de/bibbuddy/IsbnRetriever.java

```diff
@@ -26,7 +26,7 @@
    * @author Luis Moßburger
    */
   IsbnRetriever(String isbn) {
-    this.isbn = isbn;
+    this.isbn = isbn.replaceAll("-", "").replaceAll("\\s","");
```

Comment:
```suggestion
    this.isbn = isbn.replaceAll("-", "").replaceAll("\\s", "");
```
There was a white space missing after the last ",".

Reply:
Done.